catkin_ws/src/state_estimation/src/sensors.py

```diff
@@ -1,231 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
why are you rermoving this?

Reply:
deleting the whole file i thought

Reply:
no but like, why are you removing the file? Would this not be useful for localization?

Reply:
nope its a full replacement for state estimation. this is why i mentioned before that all the changes you were doing in state estimation could be wasted effort